src/NHSD.BuyingCatalogue.Ordering.Api/Extensions/ClaimsPrincipalExtensions.cs

```diff
@@ -1,25 +1,26 @@
-﻿using System;
-using System.Security.Claims;
-
-namespace NHSD.BuyingCatalogue.Ordering.Api.Extensions
-{
-    internal static class ClaimsPrincipalExtensions
-    {
-        private const string PrimaryOrganisationIdType = "primaryOrganisationId";
-
-        public static Guid GetPrimaryOrganisationId(this ClaimsPrincipal user)
-        {
-            return new Guid(user.FindFirst(PrimaryOrganisationIdType).Value);
-        }
-
+﻿using System;
+using System.Security.Claims;
+using NHSD.BuyingCatalogue.Ordering.Domain;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.Extensions
+{
+    internal static class ClaimsPrincipalExtensions

```

Comment:
Does this changeset just involve line endings? Were they inconsistent?

Reply:
I noticed some files were LF so i converted them to CRLF  as i thought we wanted consistent line ending.

Reply:
Brilliant 👍